packages/appstore/src/modules/trading-hub/index.tsx

```diff
@@ -284,9 +283,8 @@ const TradingHub: React.FC = () => {
                 <JurisdictionModal context={store} openPasswordModal={openRealPasswordModal} />
                 <CFDPasswordModal context={store} platform={platform} />
                 <CFDDbviOnBoarding context={store} />
-                <CFDPersonalDetailsModal context={store} />
                 <CFDResetPasswordModal context={store} platform={platform} />
-                <CFDServerErrorDialog context={store} />
+                <CFDServerErrorDialog />
```

Comment:
for now we reverted back the context due to this commit: https://github.com/binary-com/deriv-app/pull/7311/commits/6a5293c57c2e2c030a6db2fe67d688afb57b1bf7

Reply:
@mahdiyeh-deriv , we still pass context as a prop to `CFDServerErrorDialog`. while removing context i am getting below error. 
_Property 'context' is missing in type '{}' but required in type 'CFDServerErrorDialogProps'.

could you pls advice?

Reply:
we need to pass the context, we will use the farzin's approach later on, for now we should pass context same as before